File: src/main/kotlin/com/jsol/mobileinfo/config/exception/ForbiddenException.java
package com.jsol.mobileinfo.config.exception;

import com.jsol.mcall.service.AccountServiceImpl;

public class ForbiddenException extends BasicException {

    public ForbiddenException(String message){
        super(403, "forbidden", message);
    }

    public ForbiddenException() {
        this("권한이 없습니다." + AccountServiceImpl.getAccountFromSecurityContext().getRole());
    }
}

File: src/main/kotlin/com/jsol/mobileinfo/config/exception/ForbiddenException.kt
package com.jsol.mobileinfo.config.exception


class ForbiddenException @JvmOverloads constructor(
    //TODO 변경하기
    message: String = "권한이 없습니다.",
//        + AccountServiceImpl.getAccountFromSecurityContext().getRole()
) : BasicException(403, "forbidden", message)
